### PR TITLE
changes for vue cli 4

### PR DESCRIPTION
**Pull request by saravankumarpa**
*Issue #:*
https://github.com/bcgov/entity/issues/2171

*Description of changes:*

The changes should have been only package.json ideally.
But with the vue cli 4.1 , they removed a behavior which we relying on .They dont prepend slashes .So we have to add slashes in the process.env .That brought in changes in some other files.

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/package.json [hunk 1]
```diff
@@ -55,10 +55,10 @@
     "sinon": "^6.3.5",
     "stylus": "^0.54.7",
     "stylus-loader": "^3.0.2",
-    "ts-jest": "^23.0.0",
-    "typescript": "^3.2.1",
+    "ts-jest": "23.0.0",
+    "typescript": "~3.5.3",
```
Comment (sutherlanda) [inline auth-web/package.json]: Not going up to 3.6 or 3.7?

Comment (saravankumarpa) [reply auth-web/package.json]: let me give a try with 3.7 ..if it works , its good to go with 3.7 ... this is the version which vue cli proposes.. But 3.7 is great..Let me give a try

Comment (saravankumarpa) [reply auth-web/package.json]: I think we used it already.so will just work.. I will just try an optional chaining feature just to see how it goes

Comment (sutherlanda) [reply auth-web/package.json]: Awesome

**Pull request merged**